### DISCUSSION TITLE
StrongSwan Disconnect Site to Site VPN after some time

### DISCUSSION THREAD

**Discussion by Sumitsrjhs** [strongswan/strongswan#809] [Q&A]
HI 

I have made site 2 site connection but VPN disconnects after some time and I need to reset the connect again and again

my ipsec.conf is 

config setup
        nat_traversal=yes
        oe=off
        protostack=netkey
        fragicmp=no

conn Office
        left=172.x.x.x
        leftsourceip=65.x.x.x
        leftsubnet=172.x.x.x/24
        leftid=65.x.x.x
        right=125.x.x.x
        rightid=125.x.x.x
        esp=aes256-sha1-modp2048
        ike=aes256-sha1-modp2048
        authby=psk
        keyexchange=ikev2
        rightsubnet=192.x.x.x/24
        auto=start
        ikelifetime=1h
        lifetime=8h
        dpddelay=30
        dpdtimeout=120
        dpdaction=restart

Please suggest

**Comment by tobiasbrunner** [comment]
Please see #196.
